src/components/AeInputAmount.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="input-group">
+    <input
+      v-model="amount"
+      v-bind="$attrs"
+      type="number"
+      :min="min"
+      :step="step"
+      placeholder="Amount"
+      class="form-control"
+      aria-label="Default"
+      aria-describedby="inputGroup-sizing-mn"
+      @input="$emit('input', $event.target.value)"
+    >
+    <div class="input-group-append">
+      <span class="input-group-text append__ae">
+        <span class="ae">AE&nbsp;</span>
+        <fiat-value
+          :display-symbol="true"
+          :amount="amount.toString()"
+        />
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import FiatValue from './FiatValue.vue';
+
+export default {
+  name: 'AeInputAmount',
+  components: {
+    FiatValue,
+  },
+  props: {
+    min: { type: Number, default: 0 },
+    step: { type: Number, default: 0.1 },
```

Comment:
```suggestion
    step: { type: Number, default: 0.1 },
    value: { type: Number, required: true },
```

Reply:
Isn't it better to use `type: Number`? Also change the type of `amount` in the parent view (which is currently String)